### ISSUE TITLE
automatic position of reveal on small screens

### ISSUE THREAD

**Issue by DanielRuf** [zurb/foundation-sites#10662]
We use the reveal with the full class and so far it looks good in the mobile preview in Chrome on desktop. But when we tested it on a Nexus 5 the reveal had a gap at the top and started 51px beneath the start of the body/html element.

**Comment by mattbag** [comment]
Yes I had something like that too. Temporary solved upon creating a new class, and adding the scss mixins for reveal and reveal full.

**Comment by rafibomb** [comment]
Isn't the reveal already full screen on mobile devices? What happens when you use no `.full` class? What is your use-case, are you making it full screen an all sizes?

**Comment by DanielRuf** [comment]
> Isn't the reveal already full screen on mobile devices? What happens when you use no .full class? What is your use-case, are you making it full screen an all sizes?

There is no real difference as it seemed.

Related: https://github.com/zurb/foundation-sites/pull/9763

**Comment by rafibomb** [comment]
Yup - that makes sense and thanks for the link. Looks like @coreysyms offered a helpful solution that people can try.

**Comment by DanielRuf** [comment]
Generally the generated `top` is still the issue. I might check and propose a PR then.
